packages/rancher-alerting-drivers/charts/templates/_helpers.tpl

```diff
@@ -1,3 +1,29 @@
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Windows cluster will add default taint for linux nodes,
+add below linux tolerations to workloads could be scheduled to those linux nodes
+*/}}
+
+{{- define "linux-node-tolerations" -}}
+- key: "cattle.io/os"
+  value: "linux"
+  effect: "NoSchedule"
+  operator: "Equal"
+{{- end -}}
+
+{{- define "linux-node-selector" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+beta.kubernetes.io/os: linux
+{{- else -}}
+kubernetes.io/os: linux
+{{- end -}}
+{{- end -}}
```

Comment:
I know this isn't the purpose for this PR, but could we add these tolerations and nodeSelectors across all the deployments in alerting drivers? I just realized Alerting Drivers today is not deployable in a Windows cluster, so a user of Rancher Monitoring V2 for Windows is ineligible to use it.

Do we have an issue tracking this? cc: @cbron @sirredbeard

Reply:
> could we add these tolerations and nodeSelectors across all the deployments in alerting drivers? 

Yes, those are already added to all deployments in alerting drivers.

Reply:
Whats point of adding these here ? They already exist in the lower charts: https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-sachet/rancher-sachet/1.0.100/templates/_helpers.tpl
If we are adding to unify then shouldn't we remove those ?

Reply:
I added them because they are used by the patch-sa Job in the newly-added hardened.yaml file. 
There is a trick to use the sub-chart's template in the main chart if you want me to remove them from the main chart.

Reply:
Oh right, no thats ok.